src/v0/destinations/slack/transform.js

```diff
@@ -208,7 +208,13 @@ const processTrack = (message, destination) => {
 
   logger.debug('templateInputTrack: ', templateInput);
 
-  const resultText = eventTemplate(templateInput);
+  let resultText;
+  try {
+    resultText = eventTemplate(templateInput);
+  } catch (err) {
+    throw new ConfigurationError('Something is wrong with the event template');
```

Comment:
Include the template in the error message.

Reply:
Please add the original error message if it can help.

Reply:
- template added.
- the original message is not very informative `(lookupProperty(...) || (depth0 && lookupProperty(...)) || alias2).call is not a function`